docs/using/image-management.md

```
@@ -0,0 +1,284 @@
+
+!!! note
+
+    The manifest is refreshed from the remote URL in the background
+    approximately once per hour. A Control Plane restart is not required for
+    routine manifest updates.
+
+### Image Tag Format
+
+pgEdge image tags follow this structure:
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)